Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/FeatureClassUtils.cs

```diff
@@ -44,7 +44,7 @@ public void ClearExistingDataset(string featureClassName)
       schemaBuilder.Build();
       MapView.Active.Redraw(true);
     }
-    catch (Exception ex) when (!ex.IsFatal()) //(GeodatabaseTableException)
+    catch (GeodatabaseTableException ex)
     {
       // "The table was not found." | System.InvalidCast
```

Comment:
should this also be handling `System.InvalidCastException`?

Reply:
looks legit with your changes already, we don't seems to cast anything here, no?

Reply:
Doesn't look to be. But there was this comment so I thought I'd ask.

Perhaps `GetDefinition<FeatureClassDefinition>` may throw if there is something in the db with the name, but it isn't a `FeatureClassDefinition`

Not sure if that's even a possibility.